packages/server/src/with-blitz.ts

```diff
@@ -24,6 +24,7 @@ export function withBlitz(nextConfig: any) {
         } else {
           config.module = config.module || {}
           config.module.rules = config.module.rules || []
+          config.module.rules.push({test: /db/, use: {loader: "null-loader"}})
```

Comment:
This makes sure that anything in the `db` folder is never included in the frontend bundle.